setup.py

```diff
@@ -32,7 +32,7 @@ def package_data(pkg, roots):
     install_requires=[
         'XBlock',
         'bleach',
-        'html-xblock~=0.1.1',
+        'html-xblock~=0.2.0',
```

Comment:
@taniwha, this should also be updated once the new version of `xblock-html` is tagged and available on PyPI.